server/storage/options.go

```diff
@@ -0,0 +1,22 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package storage
+
+//Options contains configuration for plugins
+type Options struct {
+	Endpoint       string
```

Comment:
Endpoint is not enough, plz add more options in net PR, like TLSconfig,

Reply:
Could I add the necessary config when I refactor the logic in api layer here?  Because I cannot make it clear which config is need for server/ms, server/auth, server/dep exactly.